firmware/defmt-rtt/Cargo.toml

```diff
@@ -8,8 +8,8 @@ license = "MIT OR Apache-2.0"
 name = "defmt-rtt"
 readme = "README.md"
 repository = "https://github.com/knurling-rs/defmt"
-version = "0.3.0"
+version = "0.3.1"
 
 [dependencies]
 cortex-m = "0.7"
-defmt = { version = "0.3", path = "../../defmt" }
```

Comment:
On a more philosophical note (aka. I am not sure myself what the best option is): Since `defmt-rtt` doesn't pin the version of `defmt` we are not required to bump it, afaik, but it will just pick up more recent versions. The question therefore is, if we should do that, or just leave it at `0.3.0` until `0.4.0`.

Reply:
In the end this doesn't really matter, it just occurred to me when seeing it.

Reply:
I would just leave it, until there is a change that defmt-rtt requires in order to function correctly.

Reply:
Makes sense. Are you going to revert it then?

Reply:
You need to revert anyway, because there is no `defmt 0.3.1`